### DISCUSSION TITLE
Aula sobre Constantes

### DISCUSSION THREAD

**Discussion by ddcsilva** [balta-io/2801#33] [Perguntas e Respostas] [answered]
Olá!
Estou acompanhando as aulas e notei que na aula sobre constantes, são informados dois exemplos, uma constante sem valor e outra com valor. E é informado que é correto e que inicializa com zero. Porém o compilador reclama, informando que a constante deve ser inicializada.

![image](https://user-images.githubusercontent.com/37319552/151465096-7c629e36-0173-48de-a610-38a229e33d37.png)

Está correto ou é alguma configuração que fiz de diferente em minha IDE? Ou talvez a versão do .NET, estou utilizando a 5

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @ddcsilva, como vai?

Esta foi uma mudança que veio no C# 10/.NET 6. Basta inicializar sua constante (Forma correta) para que o Warning seja removido.

**Reply by ddcsilva** [reply]
Obrigado pela explicação @andrebaltieri!
